src/KeyVault/KeyVault/Commands/NewAzureKeyVault.cs

```diff
@@ -195,6 +248,23 @@ public override void ExecuteCmdlet()
                     WriteWarning(Resources.VaultNoAccessPolicyWarning);
                 }
             }
+            else
+            {
+                
+                string whatIfMessage = null;
+                string warningMessage = null;
+                string captionMessage = null;
+                PSWhatIfOperationResult whatIfResult = this.ExecuteWhatIf(VaultCreationParameter);
+                string whatIfFormattedOutput = WhatIfOperationResultFormatter.Format(whatIfResult);
+                string cursorUp = $"{(char)27}[1A";
```

Comment:
```suggestion
                string cursorUp = $"{(char)27}[1A";
```
Looks like a magic number.